src/utils/siteColorUtils.js

```diff
@@ -134,10 +134,9 @@ const getSiteColors = async (siteName) => {
     const settingsResp = await axios.get(
       `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/settings`
     )
```

Comment:
Also using the v1 endpoint

Reply:
> PR looks okay - but we should swap out the v1 endpoints for settings for the v2 endpoints. Apart from the commented places, we also need to swap them out in lines 118 and 356 in the Settings layout file

Fixed with [4117e66](https://github.com/isomerpages/isomercms-frontend/pull/618/commits/4117e66f1834aaeaa427d0168a9d235a3f30c589)